contracts/settings/ATMSettings.sol

```diff
@@ -53,14 +59,71 @@ contract ATMSettings is IATMSettings {
 
     /* Constructor */
 
-    constructor(address settingsAddress) public {
-        require(settingsAddress != address(0x0), "SETTINGS_MUST_BE_PROVIDED");
+    constructor(
+        address settingsAddress,
+        address atmTokenLogicAddress,
+        address atmGovernanceLogicAddress
+    ) public {
+        require(settingsAddress.isContract(), "SETTINGS_MUST_BE_A_CONTRACT");
```

Comment:
Do we need to validate atmTokenLogicAddress and atmGovernanceLogicAddress are contracts?